python/mxnet/ndarray/numpy/_op.py

```diff
@@ -178,6 +179,68 @@ def minimum(x1, x2, out=None):
     return _ufunc_helper(x1, x2, _npi.minimum, _np.minimum, _npi.minimum_scalar, None, out)
 
 
+@set_module('mxnet.ndarray.numpy')
+def mean(a, axis=None, dtype=None, out=None, keepdims=False):  # pylint: disable=arguments-differ
```

Comment:
the order of param  is right?

Reply:
@mikemwx Please double-check and make sure the order of parameters is correct.

Reply:
Yes, I have wrapped it